### ISSUE TITLE
Be able to output sound over 100%

### ISSUE THREAD

**Issue by NYYR1KK1** [openMSX/openMSX#1161]
Hi,

I've noticed that for some weird reason the volume settings are artificially limited to 100% I have now an issue that I have to record audio from openMSX first and then play it back in VLC that allows sound playback on volume of 125%... It works but is unnecessary extra step.

Also if I would like to emulate ie. original Spectravideo SVI-738 PSG sound volume I would probably need to use something like 'set PSG_volume 200' to match clipping with original hardware, but ATM this does not seem to work. I understand these limits are ok in GUI, but could you please consider removing these limitations at least from console?

**Comment by MBilderbeek** [comment]
For the first thing: why do you need more than 100%? Did you also try to increase the master_volume setting?

For the 2nd thing: internally openMSX will clip if the volume gets too large, if we would allow more than 100 for the volume. But it's not likely the clipping will happen in the same way as on a real MSX... Did you also try to adjust the volume in the device itself, via its XML file? That's more fitting to the situation (the hardware itself is causing the loud volume).

**Comment by NYYR1KK1** [comment]
Yes, I've put everything to 100% The need comes from the fact I try to transfer files trough 3.5mm headphone connector, but my potato does not output strong enough volume unless I save to WAV and digitally make it louder... It is not lacking much, but just barely enough to fail.

The XML-tip was good, I have to try that.

**Comment by MBilderbeek** [comment]
@NYYR1KK1 is this issue solved or still relevant?
